includes/update.php

```diff
@@ -70,6 +72,38 @@ public function __construct( $version = SIMPLE_CALENDAR_VERSION ) {
 		$this->installed_ver = is_null( $installed ) ? get_option( 'gce_version', null ) : $installed;
 		$this->new_ver = $version;
 
+		/**
+		 * One time update of some calculated post meta data that is
+		 * expensive to do on calendar admin page custom column data output..
+		 *
+		 * We're going to store the events source, calendar type, and
+		 * shortcode data from /wp-admin/edit.php?post_type=calendar table
+		 * in post meta for quicker access.
+		 *
+		 * This will also be done when new calendars are created/saved/updated/etc.
+		 */
+		if ( false === get_option( '_simcal_calendar_post_meta_update_complete' ) ) {
+
+			if ( version_compare( $this->installed_ver, $this->new_ver, '>=' ) ) {
+				$calendar_posts = get_posts(
+					array(
+						'post_type' => 'calendar',
+						// Setting to 200 posts. Should cover most installations.
```

Comment:
What if the installation has more than 200?

Reply:
200 calendars seems like a lot to me. Thoughts on a limit or just grab all and set up a function to batch process them?

Reply:
As of right now what happens if the user has 300 calendars? Would the 100 remaining calendars be broken since they didn't get caught by this upgrade routine?

Reply:
@zachwills that's correct. this is the one issue I didn't address in last night's code updates. I'm going to create a function for this to batch process the update, including all calendars.